src/app/ApplicationTemplate.Shared/Presentation/Settings/SettingsPageViewModel.cs

```diff
@@ -10,13 +10,30 @@
 using Chinook.StackNavigation;
 using MessageDialogService;
 using Microsoft.Extensions.Localization;
+using Windows.UI.Xaml;
 using Xamarin.Essentials;
 using Xamarin.Essentials.Interfaces;
 
 namespace ApplicationTemplate.Presentation
 {
 	public partial class SettingsPageViewModel : ViewModel
 	{
+
+		public SettingsPageViewModel()
+		{
+			AddDisposable(this.GetProperty(x => x.IsThemeChangeEnabled)
+				.Observe()
+				.SelectManyDisposePrevious((e, ct) => OnThemeSwitchButtonChange())
```

Comment:
Use commands when you want to implement commands.
You should very rarely use `SelectManyDisposePrevious`.

Reply:
FYI you can invoke a command from a ToggleSwitch like this.

```xaml
<ToggleSwitch IsOn="{Binding SomeBoolean, Mode=TwoWay}">
	<interactivity:Interaction.Behaviors>
		<core:EventTriggerBehavior EventName="Toggled">
			<core:InvokeCommandAction Command="{Binding SomeCommand}" />
		</core:EventTriggerBehavior>
	</interactivity:Interaction.Behaviors>
</ToggleSwitch>
```